### PR TITLE
Compare the model with experimental results

### PR DESCRIPTION
**Pull request by AmelieLaurens**
-Add experimental values in the deck.yaml
-Add a class Experimental to get the experimental values
-Add the values used by Mellado to compare the model with experiment in his article in the deck.yaml
-Add variables to get Mellado' values in the Polymer and RJSMachine
-Add a class Comparison to compare the model with experiments and save the graphic in the folder graphics

### CODE REVIEW

**Comment by ilyasst** [comment]
@jaklengaigne Bump !

File: rotary_jet_spinning/experimental/experimental_values.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+
+class Experimental:
+
```
Comment (jaklengaigne) [inline rotary_jet_spinning/experimental/experimental_values.py:6]: This new class warrants a little description for its purpose.

File: rotary_jet_spinning/comparison_with_exp/comparison.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+    def comparison_with_experiment(self, deck, polymer, machine, model, features, experimental):
+        
+        # X Axis : the scaling factor
+        scaling_factor = numpy.linspace(1.0, machine.maximum_angular_velocity/machine.mellado_angular_velocity, features.discretisation)
+        
+        # Y Axis : radius ratio
+        radius_ratio = []
+        for k in range(features.discretisation):
```
Comment (jaklengaigne) [inline rotary_jet_spinning/comparison_with_exp/comparison.py:43]: This has to be explained somewhere. I suggest adding an explanation into the Mellado_equation.md

**Review by jaklengaigne — APPROVED** [review]
The code runs smoothly.
While the code is ready to use other experimental data, it may need more adaptations to do so. To be determined in due time.
We should also think of collecting literature data from other articles to do a experimental database.

**Pull request merged**